python/packages/azure-ai/agent_framework_azure_ai/_chat_client.py

```diff
@@ -898,6 +899,9 @@ async def _prep_tools(
                                 filter=additional_props.get("filter", ""),
                             )
                             tool_definitions.extend(ai_search.definitions)
+                case OpenApiToolDefinition():
```

Comment:
We should do this:
```suggestion
                case ToolDefinition():
```
Then it works across all current and future tools.